r2i/tensorrt/Makefile.am

```diff
@@ -0,0 +1,56 @@
+# Copyright (C) 2018-2020 RidgeRun, LLC (http://www.ridgerun.com)
+# All Rights Reserved.
+#
+# The contents of this software are proprietary and confidential to RidgeRun,
+# LLC.  No part of this program may be photocopied, reproduced or translated
+# into another programming language without prior written consent of
+# RidgeRun, LLC.  The user is free to modify the source code after obtaining
+# a software license from RidgeRun.  All source code changes must be provided
+# back to RidgeRun without any encumbrance.
+
+if HAVE_TENSORRT
+
+noinst_LTLIBRARIES = libtensorrt.la
+
+# Define a custom area for our headers
+tensorrtincludedir = @R2IINCLUDEDIR@/r2i/tensorrt
+
+libtensorrt_la_SOURCES =         \
+    engine.cc                    \
+    frame.cc                     \
+    frameworkfactory.cc          \
+    loader.cc                    \
+    model.cc                     \
+    parameters.cc                \
+    prediction.cc
+
+libtensorrt_la_CPPFLAGS =        \
+    $(RR_CPPFLAGS)               \
+    $(CODE_COVERAGE_CPPFLAGS)    \
+    -I$(CUDA_INCDIR)
```

Comment:
Don't we have CUDA_CFLAGS?

Reply:
yes, but they add some flags that appear to be meant for nvcc rather than g++>